### PR TITLE
Completed javascript-functions - Lisa Nguyen

### PR DESCRIPTION
**Pull request by lisa-t-nguyen**
image

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  firstName: 'Lisa',
+  lastName: 'Nguyen'
+};
+
+var getFirstNameResult = getFirstName(person);
+
+console.log('getFirstNameResult', getFirstNameResult);
+
+function getLastElement(array) {
+  return array[3];
```
Comment (Zhiwutian) [inline javascript-functions/main.js:43]: this should be done using the `array.length-1` so you don't have to hard code a number in

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

**Comment by lisa-t-nguyen** [comment]
image

Updated screenshot

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**